gneiss/regression/tests/test_ols.py

```diff
@@ -15,334 +15,234 @@
-            index=['Y1', 'Y2']).T
-        self.tree = TreeNode.read(['(c, (b,a)Y2)Y1;'])
-        self.unannotated_tree = TreeNode.read(['(c, (b,a));'])
-        self.metadata = pd.DataFrame({
-            'lame': [1, 1, 1, 1, 1],
-            'real': [1, 2, 3, 4, 5]
-        }, index=['s1', 's2', 's3', 's4', 's5'])
-
         np.random.seed(0)
-        n = 15
-        a = np.array([1, 4.2, 5.3, -2.2, 8])
-        x1 = np.linspace(.01, 0.1, n)
-        x2 = np.logspace(0, 0.01, n)
-        x3 = np.exp(np.linspace(0, 0.01, n))
-        x4 = x1 ** 2
-        self.x = pd.DataFrame({'x1': x1, 'x2': x2, 'x3': x3, 'x4': x4})
-        y = (a[0] + a[1]*x1 + a[2]*x2 + a[3]*x3 + a[4]*x4 +
-             np.random.normal(size=n))
-        sy = np.vstack((y, y/10)).T
-        self.y = pd.DataFrame(ilr_inv(sy), columns=['a', 'b', 'c'])
-        self.t2 = TreeNode.read([r"((a,b)y1,c)y0;"])
-
-        # for testing the plugins
+        b01, b11, b21 = 1, 2, -3
+        b02, b12, b22 = 2, -1, 4
+        n = 50
+        x1 = np.linspace(0, 10, n)
+        x2 = np.linspace(10, 15, n)
+        e = np.random.normal(size=n) * 10
+        y1 = b01 + b11 * x1 + b21 * x2 + e
+        e = np.random.normal(size=n) * 10
+        y2 = b02 + b12 * x1 + b22 * x2 + e
+        Y = pd.DataFrame(np.vstack((y1, y2)).T,
+                         columns=['y1', 'y2'])
+
+        B = pd.DataFrame([[b01, b11, b21],
+                          [b02, b12, b22]])
+
+        X = pd.DataFrame(
+            np.vstack((np.ones(n), x1, x2)).T,
+            columns=['Intercept', 'x1', 'x2'])
+
+        self.Y = Y
+        self.B = B
+        self.X = X
+        self.r1_ = smf.OLS(endog=y1, exog=X).fit()
+        self.r2_ = smf.OLS(endog=y2, exog=X).fit()
+        self.tree = TreeNode.read(['(c, (b,a)y2)y1;'])
+
         self.results = "results"
         os.mkdir(self.results)
 
-        self.table2 = pd.DataFrame({
-            's1': A([1., 1.]),
-            's2': A([1., 2.]),
-            's3': A([1., 3.]),
-            's4': A([1., 4.]),
-            's5': A([1., 5.]),
-            's6': A([1., 6.]),
-            's7': A([1., 7.]),
-            's8': A([1., 8.]),
-            's9': A([1., 9.]),
-            's10': A([1., 10.]),
-            's11': A([1., 11.]),
-            's12': A([1., 12.]),
-            's13': A([1., 13.]),
-            's14': A([1., 14.]),
-            's15': A([1., 15.])},
-            index=['Y1', 'Y2']).T
-
-        self.metadata2 = pd.DataFrame({
-            'lame': [1, 1, 1, 1, 1,
-                     1, 1, 1, 1, 1,
-                     1, 1, 1, 1, 1],
-            'real': [1, 2, 3, 4, 5,
-                     6, 7, 8, 9, 10,
-                     11, 12, 13, 14, 15]
-        }, index=['s1', 's2', 's3', 's4', 's5',
-                  's6', 's7', 's8', 's9', 's10',
-                  's11', 's12', 's13', 's14', 's15'])
-
     def tearDown(self):
         shutil.rmtree(self.results)
 
+    def test_ols_test(self):
 
-class TestOLSFunctions(TestOLS):
```

Comment:
Are all these test really not needed? Where did these test move? For example, we don't have test_ols or test_ols_table_immutable in the new code ...